sympy/printing/tests/test_lambdarepr.py

```diff
@@ -99,8 +99,7 @@ def test_piecewise():
     )
     l = lambdarepr(p)
     eval(h + l)
-    assert l == "((1) if (x >= 1) else (2) if (x >= 2) else (3) if (x >= 3)"\
-                        " else (4) if (x >= 4) else (5) if (x >= 5) else (6))"
+    assert l == "((1) if (x >= 1) else (6))"
```

Comment:
It would be better to change the test to `<=` rather than rewrite it like this. I think that the way you've rewritten this misses the original intention of the tests.

Reply:
@oscarbenjamin  Hi! I could not understand your suggestion. I modified the function to do this exactly. So it should give this result. I think there is another test which has the `<` sign too? Please clarify me if I am wrong.

Reply:
Okay I see what you mean. In that case reorder the conditions so it starts with x>=5.

Reply:
Please don't mark the conversation as resolved when it isn't resolved.